<?xml version="1.0" encoding="UTF-8"?>
<!--
  - Copyright (c) 2022 船山信息 chuanshaninfo.com
  - The project is licensed under Mulan PubL v2.
  - You can use this software according to the terms and conditions of the Mulan
  - PubL v2. You may obtain a copy of Mulan PubL v2 at:
  -          http://license.coscl.org.cn/MulanPubL-2.0
  - THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
  - EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
  - MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
  - See the Mulan PubL v2 for more details.
  -->

<ui version="4.0">
    <class>OPlayerController</class>
    <widget class="QWidget" name="OPlayerController">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>575</width>
                <height>70</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QWidget" name="controller" native="true">
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>60</height>
                        </size>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QToolButton" name="m_playBtn">
                                <property name="styleSheet">
                                    <string notr="true">:hover{
                                        background:none;
                                        }
                                    </string>
                                </property>
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="icon">
                                    <iconset resource="../../resources.qrc">
                                        <normaloff>:/resources/icon/player_play.png</normaloff>
                                        :/resources/icon/player_play.png
                                    </iconset>
                                </property>
                                <property name="checkable">
                                    <bool>true</bool>
                                </property>
                                <property name="checked">
                                    <bool>false</bool>
                                </property>
                                <property name="popupMode">
                                    <enum>QToolButton::DelayedPopup</enum>
                                </property>
                                <property name="toolButtonStyle">
                                    <enum>Qt::ToolButtonIconOnly</enum>
                                </property>
                                <property name="autoRaise">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QSlider" name="m_playSlider">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="m_time_current">
                                <property name="text">
                                    <string>00:00:00</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="m_time_split">
                                <property name="text">
                                    <string>/</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="m_time_total">
                                <property name="text">
                                    <string>00:00:00</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QToolButton" name="m_muteBtn">
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="icon">
                                    <iconset resource="../../resources.qrc">
                                        <normaloff>:/resources/icon/player_mute.png</normaloff>
                                        :/resources/icon/player_mute.png
                                    </iconset>
                                </property>
                                <property name="autoRaise">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QSlider" name="m_volumeSlider">
                                <property name="maximumSize">
                                    <size>
                                        <width>120</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="../../resources.qrc"/>
    </resources>
    <connections/>
</ui>
